core/store/src/db/rocksdb.rs

```diff
@@ -382,14 +382,46 @@ fn rocksdb_read_options() -> ReadOptions {
 fn rocksdb_block_based_options(
     block_size: bytesize::ByteSize,
     cache_size: bytesize::ByteSize,
+    col: DBCol,
 ) -> BlockBasedOptions {
     let mut block_opts = BlockBasedOptions::default();
+    block_opts.set_checksum(/*no checksum */ '\0');
```

Comment:
```suggestion
    // request no checksumming
    block_opts.set_checksum('\0');
```

The previous style feels too much like C / C++ and not rusty 😃